hacks/bump-jenkins-version/bump-jenkins.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+#https://updates.jenkins-ci.org/download/war/2.73.1/jenkins.war
+
+JENKINS_DIST_GIT=jenkins
+
+usage() {
+    echo >&2
+    echo "Usage `basename $0` <jenkins-war-version> <branch>" >&2
+    echo >&2
+    echo " Example: ./bump-jenkins.sh 2.73.1 rhaos-3.7-rhel-7" >&2
+    exit 1
+}
+
+# clone jenkins distgit
+# switch to latest branch 
+setup_dist_git() {
+  container="jenkins"
+  workingdir=$(dirname $(realpath $0))/working
+
+  if ! klist &>${workingdir}/logs/${container}.output ; then
+    echo "Error: Kerberos token not found." ; popd &>${workingdir}/logs/${container}.output ; exit 1
+  fi
+
+  cd ${workingdir}
+  rhpkg ${USER_USERNAME} clone "${container}" &>${workingdir}/logs/${container}.output
```

Comment:
won't you need to cd into this directory for some of your other rhpkg invocations to work?

Reply:
✔️